lib/hotel.rb

```diff
@@ -0,0 +1,209 @@
+require 'date'
+require_relative 'reservation'
+require 'awesome_print'
+require 'pry'
+
+module Admin
+  class Hotel
+    attr_reader :rooms
+
+    def initialize(rooms)
```

Comment:
Consider renaming this variable to `room_count` or something since right now it isn't clear what this variable would contain